status-im/status-legacy
# Remove web3

fix #8929 
fix #8954

We can finally get rid of web3

##### Functional

- wallet / transactions

### Steps to test

The PR is just removing unused dependencies and replace the last use of web3 to create BigNumbers by using the underlying BigNumber.js library directly. So I don't expect any change of behavior.

Tested on Android simulator

- wallet should display same numbers as nightly

status: ready
